README.md

```diff
@@ -84,13 +84,27 @@ The CLI is available by using the command `jsipfs` in your terminal. This is ali
 
 The code published to npm that gets loaded on require is in fact a ES5 transpiled version with the right shims added. This means that you can require it and use with your favourite bundler without having to adjust the asset management process.
 
+```js
+var ipfs = require('ipfs');
+```
+
 ### Use in a browser using a script tag
 
-Simply include ipfs in your bundled javascript file.
+Loading this module in a browser (using a `<script>` tag) makes the `Ipfs` object available in the global namespace.
 
-```html
-<script type="text/javascript" src="assets/bundled.js"></script>
-```
+The last published version of the package becomes available on [npmcdn](https://npmcdn.com/) and thus you may use it as the source:
```

Comment:
The homepage of https://npmcdn.com/ currently says the following:

> You may also use a [tag](https://docs.npmjs.com/cli/dist-tag) or [version range](https://docs.npmjs.com/misc/semver) instead of a fixed version number, or omit the version/tag entirely to use the `latest` tag.

And that's why I've decided to omit the version/tag.